ports/rp2/machine_pwm.c

```diff
@@ -87,16 +139,40 @@ STATIC mp_obj_t mp_machine_pwm_make_new(const mp_obj_type_t *type, size_t n_args
     uint slice = pwm_gpio_to_slice_num(gpio);
     uint8_t channel = pwm_gpio_to_channel(gpio);
     machine_pwm_obj_t *self = &machine_pwm_obj[slice * 2 + channel];
+    self->invert = 0;
     self->duty_type = DUTY_NOT_SET;
 
     // Select PWM function for given GPIO.
     gpio_set_function(gpio, GPIO_FUNC_PWM);
```

Comment:
This line should be removed since it causes glitches like I was describing, and you already copied it to the end of the function.

Reply:
This was fixed.